shared/src/business/useCases/courtIssuedDocument/addServedStampToDocument.test.js

```diff
@@ -12,7 +12,9 @@ describe('addServedStampToDocument', () => {
   let drawTextMock;
```

Comment:
I know from experience that writing tests for these issues is incredibly difficult. Was there any way to cover this bug with a test?

Reply:
Great question, I am not sure if this bug could have been covered by a test :( however the test suite did fail after we incorporated our bug fix and we updated it to reflect that change.

Reply:
Ok, that works! Thanks!